### PR TITLE
RS/YJ/UPDATE BASELINE SYSTEM TYPE

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR adds ```get_baseline_system_types``` function for the approved section 22 rules (1, 2, 3, 4, 5, 6, 24, and 19). I also updated System types 7 and 11.1 in the RMDs based on Juan's work to make ```get_baseline_system_types``` function identify system type correctly.

After the updates, all the rules in this PR passed as expected.

### CODE REVIEW

File: rct229/ruleset_functions/baseline_systems/baseline_system_util.py [hunk 1]
```diff
@@ -29,7 +29,10 @@ class HVAC_SYS:
     SYS_9 = "Sys-9"
     SYS_9B = "Sys-9b"
     SYS_10 = "Sys-10"
+    SYS_11B = "Sys-11b"
```
Comment (weilixu) [inline rct229/ruleset_functions/baseline_systems/baseline_system_util.py:32]: There is no SYS11B, Sub types shall be SYS11_1B...

File: rct229/ruleset_functions/baseline_systems/baseline_system_util.py [hunk 2]
```diff
@@ -40,7 +43,9 @@ class HVAC_SYS:
     SYS_12B = "Sys-12b"
     SYS_12C = "Sys-12c"
     SYS_13 = "Sys-13"
-    SYS_13A = "Sys-13A"
+    SYS_13A = "Sys-13a"
+    SYS_13B = "Sys-13b"
+    SYS_13C = "Sys-13c"
```
Comment (weilixu) [inline rct229/ruleset_functions/baseline_systems/baseline_system_util.py:48]: No SYS_13B and C (see function is_baseline_system_13)

**Review by weilixu — CHANGES_REQUESTED** [review]
Thanks for updating the rules. System label needs to be updated.

File: rct229/ruleset_functions/baseline_systems/baseline_system_util.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/baseline_systems/baseline_system_util.py:32]: Fixed.

File: rct229/ruleset_functions/baseline_systems/baseline_system_util.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/baseline_systems/baseline_system_util.py:48]: When I saw the [22-1 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section22/Rule22-1.md#:~:text=11C%22%2C%20%22SYS%2D12C%22%2C-,%22SYS%2D13C%22,-%5D)%3A%20CHECK_RULE_LOGIC), there is  ```SYS-13C```. Should I remove this in the RDS?

Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/baseline_systems/baseline_system_util.py:48]: In the [is_baseline_system_13 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_13.md), there are only ```Sys-13``` and ```Sys-13a```. 

However, in some of section 22 RDS (e.g., [22-1](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section22/Rule22-1.md#:~:text=11C%22%2C%20%22SYS%2D12C%22%2C-,%22SYS%2D13C%22,-%5D))), there is"SYS-13C".

Is it correct to follow the is_baseline_system_13 RDS and fix section 22 RDS?

**Comment by yunjoonjung-PNNL** [comment]
The system type was corrected in util, rule, and RDS. I'd appreciate it if you could review it one more time. Thank you!

**Review by charliepnnl — APPROVED** [review]
Looks fine to me

**Review by weilixu — APPROVED** [review]

**Pull request merged**